### DISCUSSION TITLE
nest.js 8 中文文档认领

### DISCUSSION THREAD

**Discussion by zuohuadong** [nestcn/docs.nestjs.cn#252] [3-常规区]
- [ ] 介绍(8/introduction)

- 概述
  - [x]  第一步 (8/firststeps)  
  - [ ] 控制器 (8/controllers) 
  - [ ] 提供者(8/providers) 
  - [ ] 模块(8/modules) 
  - [ ] 中间件(8/middlewares)
  - [ ] 异常过滤器(8/eceptionfilters)
  - [x] 管道(8/pipes) 
  - [x] 守卫(8/guards) 
  - [ ] 拦截器(8/interceptors) 
  - [ ] 自定义装饰器(8/customdecorators) 【待提交】

- [ ] 基本原理(8/fundamentals)

- [x] 技术(8/techniques)  

- [x] GraphQL(8/graphql)

- [ ] WEBSOCKETS(8/websockets)

- [x] 微服务(8/microservices) 【待提交】

- [ ] 秘籍 (8/recipes) 

- [ ] 独立应用(8/standalone-applications)

- [ ] CLI (8/cli)

- [ ] FAQ (8/faq)

- [x] 迁移指南(8/migrationguide)

- [ ] 谁在使用 Nest (8/discover)

### 回复：

比如：
我要认领翻译「基础-依赖注入」

### 署名规范：
满足以下条件之一即可：
1. 文档整体翻译
2. 提交修改累积超过3处
参考 7.x  文档进行署名（头像请勿超过 80X80 ）。

**Comment by hehu321** [comment]
我要认领翻译

**Comment by ganshiqingyuan** [comment]
我要认领微服务

**Comment by snowyYU** [comment]
我要认领翻译【概述-自定义装饰器】

**Comment by zuohuadong** [comment]
@ganshiqingyuan @snowyYU  认领成功，欢迎提交 PR 。
@hehu321   需要回复认领内容

**Comment by qianfeiqianlan** [comment]
我要认领翻译「概述-第一步 」

**Reply by zuohuadong** [reply]
认领成功，请尽快提交翻译

**Reply by qianfeiqianlan** [reply]
PR: https://github.com/nestcn/docs.nestjs.cn/pull/260

**Comment by elonglau** [comment]
我要认领翻译「GraphQL」

**Reply by zuohuadong** [reply]
认领成功

**Comment by Mingyang-Li** [comment]
我要认领翻译 [概述 - 控制器]

**Comment by renjietan** [comment]
我要认领微服务
